charts/kong/example-values/minimal-k4k8s-enterprise.yaml

```diff
@@ -0,0 +1,18 @@
+
+
+image:
+  repository: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s
+  tag: 2.0.4.1-centos
+  pullSecrets:
+    - you-must-create-a-registry-secret
+
+enterprise:
+  enabled: true
+  license_secret: you-must-create-a-license-secret
+
+env:
+  database: "off"
+
+ingressController:
+  enabled: true
+  installCRDs: false
```

Comment:
This seems wrong. You shouldn't even need `ingressController` section in any of the setups which include the controller.

Reply:
I see. Is this for Helm 3 compat? Have we considered Helm 2 vs Helm 3?

Reply:
That and in line with the earlier comment re the not quite minimal meaning of minimal here--even though this is the default, it should call out the major things it's doing explicitly so that you don't have to cross-reference.

As far as Helm 2, I'm working from the assumption that most new installs should (hopefully) be using Helm 3. I'm not really targeting Helm 2 for first-class support across the board, only where it'd break the template entirely for Helm 2. Will note as much in the examples README.

Reply:
In that case, all docs should have `helm install kong/kong --values <url>` for helm3 and then for helm2, `helm install kong/kong --values <url> --set <key>=value`

Reply:
Showing `-f <filepath>` instead: half of these won't work as-is, so I don't want to suggest otherwise with the URL.